hyperspy/tests/signal/test_eds_tem.py

```diff
@@ -157,15 +159,49 @@ def setUp(self):
 
         s.set_elements(elements)
         s.add_lines(xray_lines)
+        s.axes_manager[0].scale = 0.5
+        s.axes_manager[1].scale = 0.5
         self.signal = s
 
     def test_quant_lorimer(self):
         s = self.signal
+        method = 'CL'
         kfactors = [1, 2.0009344042484134]
+        composition_units = 'weight'
         intensities = s.get_lines_intensity()
-        res = s.quantification(intensities, kfactors)
-        nt.assert_true(np.allclose(res[0].data, np.array(
-                       [22.70779, 22.70779]), atol=1e-3))
+        res = s.quantification(intensities, method, kfactors, composition_units)
+        np.testing.assert_allclose(res[0].data, np.array([[22.70779, 22.70779],
+                    [22.70779, 22.70779]]), atol=1e-3)
+
+    def test_quant_zeta(self):
+        s = self.signal
+        method = 'zeta'
+        compositions_units = 'weight'
+        factors = [20, 50]
+        intensities = s.get_lines_intensity()
+        res = s.quantification(intensities, method, factors, compositions_units)
+        np.testing.assert_allclose(res[1].data, np.array(
+                [[ 2.7125736e-03,   2.7125736e-03],
+                [  2.7125736e-03,   2.7125736e-03]]), atol=1e-3)
+        np.testing.assert_allclose(res[0][1].data, np.array(
+                [[ 80.962287987,   80.962287987],
+                [  80.962287987,   80.962287987]]), atol=1e-3)
```

Comment:
Only if this were a real sample. I have chosen the z-factors and cross sections fairly arbitrarily because I don't know the exact ones, hence the variation in percentages seen. For a real sample they should be somewhere similar yes.